Clean Swift/Interactor.xctemplate/___FILEBASENAME___Interactor.swift

```diff
@@ -8,24 +8,13 @@
 
 import UIKit
 
-protocol ___FILEBASENAMEASIDENTIFIER___InteractorOutput {
-    func presentSomething(response: ___FILEBASENAMEASIDENTIFIER___Response)
+protocol ___VARIABLE_sceneName___InteractorOutput {
 }
 
-class ___FILEBASENAMEASIDENTIFIER___Interactor: ___FILEBASENAMEASIDENTIFIER___ViewControllerOutput {
-    var output: ___FILEBASENAMEASIDENTIFIER___InteractorOutput!
-    var worker: ___FILEBASENAMEASIDENTIFIER___WorkerProtocol!
+class ___VARIABLE_sceneName___Interactor: ___VARIABLE_sceneName___ViewControllerOutput {
```

Comment:
We are currently using `final` keyword for classes by default. For most layers in the architecture it makes perfect sense. I am just writing it here as a suggestion.

Reply:
Good idea. Updated. 😊